src/python/pants/bin/pants_runner.py

```diff
@@ -91,26 +88,17 @@ def run(self):
         ExceptionSink.reset_should_print_backtrace_to_terminal(
             global_bootstrap_options.print_exception_stacktrace
         )
-        ExceptionSink.reset_log_location(global_bootstrap_options.pants_workdir)
 
         if self._should_run_with_pantsd(global_bootstrap_options):
             try:
-                return RemotePantsRunner(
-                    self._exiter, self._args, self._env, options_bootstrapper
-                ).run()
+                RemotePantsRunner(self._exiter, self.args, self.env, options_bootstrapper).run()
+                return
             except RemotePantsRunner.Fallback as e:
-                logger.warning(
-                    "caught client exception: {!r}, falling back to non-daemon mode".format(e)
-                )
+                logger.warning(f"Client exception: {e}, falling back to non-daemon mode")
```

Comment:
This is a bad change. With logging, we intentionally use `{}` for lazy interpolation of strings. https://docs.python.org/3/howto/logging.html#optimization